go/state/cppstate/cpp_state.go

```diff
@@ -279,6 +280,10 @@ func (cs *CppState) Check() error {
 	return nil
 }
 
+func (cs *CppState) CreateWitnessProof(address common.Address, keys ...common.Key) (witness.Proof, error) {
+	panic("not implemented")
+}
```

Comment:
I think this 'not implemented' issues might get piled up - we had the same thing with `HasEmptyStorage` func and I believe we might be extending these interfaces in near future. 

How about we create a general error `ErrNotImplemented` and return it instead of panicking and we will have this prepared for future unimplemented functions within unused schemas?

Reply:
We could do this, but I wonder what would be the difference. `panic` is for sure more severe, but these methods should not be ever called. When only the public API and supported configuration is called, there is no way to execute these methods. If the client bypasses this and calls something which is not supported, I do not see a lot of difference in which way he is informed about calling an unsupported feature.

Reply:
ic. Yeah, you are right. There is actually no way this gets called by the public API. Lets keep the panic.